zomes/rea_agent/zome/src/lib.rs

```diff
@@ -47,6 +47,11 @@ fn get_agent(ByAddress { address }: ByAddress<AgentAddress>) -> ExternResult<Res
     Ok(handle_get_agent(address)?)
 }
 
+#[hdk_extern]
+fn get_revision(ByRevision { revision_id }: ByRevision) -> ExternResult<ResponseData> {
+    Ok(handle_get_revision(revision_id)?)
```

Comment:
I've been wondering if it makes any difference to write it this way, versus just: 
handle_get_revision(revision_id)

Reply:
I could never get `Error`/`Result` conversion traits to work without the unwrap/rewrap. If you are able to, I'm interested in making it more succinct :wink: